site/_apps/viper-news.txt

```diff
@@ -12,12 +12,12 @@ api: 'Hacker News Firebase API'
 hosting: 'Google App Engine'
 authors: 
     - name: 'Andrea'
-lighthouse: '93/100'
-interactive-em: '3.1s'
-interactive-faster-3g: '2.0s'
-lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170511_7Q_15882b2c52ed51c614ec707736e52940&run=3
-wpt-em-link: https://www.webpagetest.org/result/170511_AW_b2e2360d4207831ece22ab4210ab9444/
-wpt-faster-3g-link: https://www.webpagetest.org/result/170511_7Q_15882b2c52ed51c614ec707736e52940/
+lighthouse: '73/100'
+interactive-em: '2.4s'
+interactive-faster-3g: '1.7s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170611_DC_6bf5fb102603eb30b3d57dc4c25b480b&run=1
```

Comment:
I used `/top/1` 'cause that's also what I redirect to by default and what the manifest points at.

I've seen other PWAs here doing the same but if that's not "allowed", then I'd definitively drop the query-string trigger since it makes no sense as url ... right?